gateway/src/shard/processor/error.rs

```diff
@@ -1,18 +1,15 @@
 //! The error type of why errors occur in the shard processor module.
 
-use std::fmt;
-use std::str::Utf8Error;
-
+use async_tungstenite::tungstenite::{Error as TungsteniteError, Message as TungsteniteMessage};
 use flate2::DecompressError;
 use futures_channel::mpsc::TrySendError;
 #[cfg(not(feature = "simd-json"))]
```

Comment:
In `gateway/src/shard/error.rs`, `#[cfg]` lines are separate from normal imports. Should this file reflect that?

Reply:
Fixed